Makefile

```diff
@@ -0,0 +1,44 @@
+XCARGS := -workspace VelocidiSDK.xcworkspace \
+		-sdk "iphonesimulator" \
+    -destination "platform=iOS Simulator,OS=14.4,name=iPhone 11 Pro Max" \
+    GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
+
+all: clean build
+
+clean:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme VelocidiSDK  clean | xcpretty
+
+build:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme VelocidiSDK build | xcpretty
+
+test:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme VelocidiSDK test | xcpretty
+
+examples: install-examples build-objc-example build-swift-example
+
+install-examples:
+	set -o pipefail && pod install --project-directory=Examples/
+
+build-objc-example:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme ObjcExample clean build | xcpretty
+
+build-swift-example:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme SwiftExample clean build | xcpretty
+
+install:
+	pod install
+
+prerequisites:
+	.scripts/prerequisites.sh
+
+oclint:
```

Comment:
I've had some challenges running this.

```
pyenv: python3: command not found

The `python3' command exists in these Python versions:
  3.7.9

Note: See 'pyenv help global' for tips on allowing both
      python2 and python3 to be found.
make: *** [oclint] Error 127
```

```
dlopen(/usr/local/bin/../lib/oclint/rules/libTooManyMethodsRule.dylib, 1): no suitable image found.  Did find:
	/usr/local/bin/../lib/oclint/rules/libTooManyMethodsRule.dylib: code signature in (/usr/local/bin/../lib/oclint/rules/libTooManyMethodsRule.dylib) not valid for use in process using Library Validation: library load disallowed by system policy

oclint: error: cannot open dynamic library: /usr/local/bin/../lib/oclint/rules/libTooManyMethodsRule.dylib
make: *** [oclint] Error 1
```

![Screenshot 2021-03-26 at 12 14 59](https://user-images.githubusercontent.com/203170/112630273-3e9bc300-8e2d-11eb-9017-8ca6044abc8c.png)

Reply:
I don't think I ever had this problem. I tried uninstalling and installing oclint and I could not replicate this issue. I don't use `pyenv`, so it might be related to that.

Reply:
Regardless of `pyenv`, I think `python3` is required. 

The second problem is not related to python.